docs/noise.ipynb

```diff
@@ -62,196 +62,673 @@
     "</table>"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "846b32703c5c"
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import cirq\n",
+    "except ImportError:\n",
+    "    print(\"installing cirq...\")\n",
+    "    !pip install --quiet cirq\n",
+    "    print(\"installed cirq.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "c3c7725dfc15"
+   },
+   "outputs": [],
+   "source": [
+    "import cirq"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
     "id": "7d3daa944d6b"
    },
    "source": [
-    "For simulation, it is useful to have `Gate` objects that enact noisy quantum evolution.\n",
-    "\n",
-    "Cirq supports modeling noise via *operator sum* representations of\n",
-    "noise (these evolutions are also known as quantum operations, quantum\n",
-    "dynamical maps, or superoperators).  \n",
+    "For simulation, it is useful to have `Gate` objects that enact noisy quantum evolution. Cirq supports modeling noise via *operator sum* representations of noise (these evolutions are also known as quantum operations, quantum dynamical maps, or superoperators). \n",
```

Comment:
as far as I understand it, the operator sum representation is *in contrast* to superoperator representation (?). If we're doing "also known as"-es then I'd mention Kraus operators (I see they're mentioned below)

Reply:
This was a remark from the original docs I was also confused by but left in. In the context of quantum channels I think the answer to your question is yes - operator sum representations and superoperators are treated as two separate concepts. 

I guess purely mathematically, Kraus operators act on density matrices (operators) and so are technically superoperators - maybe this was the original meaning? I find the operator, superoperator, super-duper-operator hierarchy unnecessarily confusing.

Anyhow I think the remark is vague and should be clarified, but will heed to better judgement.

Reply:
For now I removed the reference to superoperators.